app/whats-new/index.js

```diff
@@ -4,8 +4,15 @@ const router = require('express').Router()
 const whatsNewContentService = require('../../common/services/whats-new-content')
 
 // Define routes
-router.get('/', async (req, res) => {
+router.get('/', async (req, res, next) => {
   const content = await whatsNewContentService.fetch()
+
+  if (content.error === 'ServerError') {
+    const error = new Error('Contentful servers are down')
+    error.statusCode = 500
+    return next(error)
+  }
```

Comment:
If we get this to raise an exception rather than return something, this code might end up slightly nicer I think.

```js
try {
  const content = await whatsNewContentService.fetch()
} catch (e) {
  return next(e)
}
```

This also means we can move the responsibility for raising the exception in the whats new content service.

Reply:
In fact, I'm not even sure we need the `catch (e) { return next(e) }` part, we could get Express to catch the exception for us and propagate it up and render the error page.